traits/Cargo.toml

```diff
@@ -12,21 +12,21 @@ targets = ['x86_64-unknown-linux-gnu']
 
 [dependencies]
 serde = { version = "1.0.111", default-features = false, features = ["derive"] }
-sp-runtime = { default-features = false, version = "5.0.0", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.17" }
-sp-std = { default-features = false, version = "4.0.0", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.17" }
+sp-runtime = { default-features = false, version = "5.0.0", git = "https://github.com/HashWarlock/substrate.git", branch = "hw-uniques-locker-trait" }
+sp-std = { default-features = false, version = "4.0.0", git = "https://github.com/HashWarlock/substrate.git", branch = "hw-uniques-locker-trait" }
 codec = { package = "parity-scale-codec", version = "2.0.0", default-features = false, features = [
 	"derive",
 ] }
 scale-info = { version = "1.0", default-features = false, features = ["derive"] }
-frame-support = { default-features = false, version = "4.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.17"}
-frame-system = { default-features = false, version = "4.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.17" }
-frame-benchmarking = { default-features = false, version = "4.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.17", optional = true }
+frame-support = { default-features = false, version = "4.0.0-dev", git = "https://github.com/HashWarlock/substrate.git", branch = "hw-uniques-locker-trait"}
+frame-system = { default-features = false, version = "4.0.0-dev", git = "https://github.com/HashWarlock/substrate.git", branch = "hw-uniques-locker-trait" }
+frame-benchmarking = { default-features = false, version = "4.0.0-dev", git = "https://github.com/HashWarlock/substrate.git", branch = "hw-uniques-locker-trait", optional = true }
 
-pallet-balances = { default-features = false, version = "4.0.0-dev", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.17" }
+pallet-balances = { default-features = false, version = "4.0.0-dev", git = "https://github.com/HashWarlock/substrate.git", branch = "hw-uniques-locker-trait" }
 
 [dev-dependencies]
-sp-runtime = { default-features = false, version = "5.0.0", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.17" }
-sp-std = { default-features = false, version = "4.0.0", git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v0.9.17" }
+sp-runtime = { default-features = false, version = "5.0.0", git = "https://github.com/HashWarlock/substrate.git", branch = "hw-uniques-locker-trait" }
+sp-std = { default-features = false, version = "4.0.0", git = "https://github.com/HashWarlock/substrate.git", branch = "hw-uniques-locker-trait" }
```

Comment:
can we switch back to `paritytech` repo yet?

Reply:
Hasn't been merged, yet. We could create a substrate repo for `rmrk-team` and put the fix there until the code is merged to `substrate`. I have the 2 PRs here https://github.com/paritytech/substrate/pull/11025
https://github.com/paritytech/cumulus/pull/1085 

I'm gonna reach out on the Matrix server to see if we can get another review.